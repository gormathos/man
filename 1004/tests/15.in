10
-6.59974 2.3436 -1.53221 1.53914 -1.70969 3.40724 9.99874 -7.70257 -1.88306 -6.98098
10
3.70402 1.21925 -2.36861 -0.437468 -3.91167 9.99453 -6.32476 3.84161 9.96171 -7.32768
