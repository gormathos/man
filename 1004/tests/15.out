1.00000 9.11953 -98.35826 -1135.04563 -100.80590 20836.98567 20903.56780 -108788.11270 -138497.30684 155595.86779 215123.72618
-1094315.54433 98811.12165 -36639.95239 130092.46574 -1154004.33116 -14331681.33381 -264499.20771 -2055067.34531 -122652831.48918 -267516.13376
-5734880.83351 -315927.71769 177858.92642 211085.12454 1105055.88220 3392263827.97638 -1277340.34232 -8338115.04450 3209756647.21128 666621.42700
